### ISSUE TITLE
How can I get each loss/xent in a batch with evaluation ?

### ISSUE THREAD

**Issue by efeiefei** [tensorflow/tensor2tensor#662]
```
    model_cls = registry.model(model_name)
    model = model_cls(hparams, mode)
    logits, losses_dict = model(features)
    loss = sum(losses_dict.values())
```
I can get avg loss for batched features with above codes.
How can I get each loss (or cross entropy) for each inputs in batch?
I have got it by changing t2t source code several months ago. But I forgot the solution...

Thanks !!!
